cli/command/run/cmd.go

```diff
@@ -234,18 +234,41 @@ func runCypress(cmd *cobra.Command, tc testcomposer.Client, rs resto.Client, as
 	rs.URL = regio.APIBaseURL()
 	as.URL = regio.APIBaseURL()
 
-	switch testEnv {
+	if testEnv != "" {
+		mode = testEnv
+	}
+
+	switch mode {
 	case "docker":
 		return runCypressInDocker(p, tc)
 	case "sauce":
 		return runCypressInSauce(p, regio, tc, rs, as)
+	case "":
+		dockerProject, sauceProject := cypress.SplitSuites(p)
+		// If there is no mode settings in config files, it should run all test suites in sauce by default.
+		if len(dockerProject.Suites) == 0 && len(sauceProject.Suites) == 0 {
+			return runCypressInSauce(p, regio, tc, rs, as)
+		}
+		if len(dockerProject.Suites) != 0 {
+			if exitCode, err := runCypressInDocker(dockerProject, tc); err != nil {
```

Comment:
This block isn't really doing anything. You can simplify it to:
```suggestion
			return runCypressInDocker(dockerProject, tc)
```

Same goes for all the other blocks.